deploy/quickstart_upstream.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
-minishift start
```

Comment:
@pleimer - FYI, after this change the script name is different, and it no longer does the `minishift start` on your behalf